.travis.yml

```diff
@@ -4,13 +4,19 @@ language: go
 
 go:
     - 1.6
+    
+env:
+    - BUILDER_VERSION=0.11.0
+
+cache:
+    directories:
+    - ./builder-cache
 
 notifications:
     email: false
 
 before_install:
-- wget https://downloads.giantswarm.io/builder/0.11.0/builder
-- chmod +x ./builder
+- cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
```

Comment:
this feels to complex.

```
test -f ./builder-cache/builder-$BUILDER_VERSION || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder -o ./builder-cache/builder-$BUILDER_VERSION
cp ./builder-cache/builder-$BUILDER_VERSION ./builder
chmod +x builder
```

this downlodas builder into the cache, if needed. No `before_cache` required.